javascript/packages/orchestrator/src/paras.ts

```diff
@@ -20,8 +20,9 @@ export async function generateParachainFiles(
   namespace: string,
   tmpDir: string,
   parachainFilesPath: string,
-  chainName: string,
+  chainName: string, //relay chain name
```

Comment:
I think we should rename the variable to `relayChainName` in order to be more clear